.dockerignore

```diff
@@ -23,7 +23,6 @@ sentry-package.json
 /coverage/
 /cover
 /build
-/dist
```

Comment:
Had to remove this due to https://github.com/GoogleContainerTools/kaniko/issues/595. Should be harmless.